addons/website_mass_mailing/views/unsubscribe_templates.xml

```diff
@@ -1,50 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
-    <template id="unsubscribe">
-        <div class="container o_unsubscribe_form">
-            <div class="row">
-                <form action="/mail/mailing/unsubscribe" method="POST" id="unsubscribe_form">
-                    <input type="hidden" name="csrf_token" t-att-value="request.csrf_token()"/>
-                    <input type="hidden" name="email" t-att-value="email"/>
-                    <input type="hidden" name="mailing_id" t-att-value="mailing_id"/>
-
-                    <div class="col-md-6 col-md-offset-3">
-
-                        <h1 class="page-header">Mailing Subscriptions</h1>
-
-                        <t t-if="contacts">
-                            <div class="alert alert-info text-center">
-                                <p>Choose your mailing subscriptions.</p>
```

Comment:
Removing this does not seem a very good idea. What's the point on having mailing lists if you must be in either all or none?

Possible solutions:

1. Remove mailing lists completely. After all, with this design it does not seem to serve any purpose.
2. Add a `website_published` button to mailing lists, so that users can choose to join it or leave it. In such case, joining adds a new contact and leaving just removes the contact from the list.

But honestly, having mailing lists and removing this form gets us in a worse situation that the one we had in v8, which was quite bad already... 😕